src/ralph/data_importer/resources.py

```diff
@@ -213,6 +215,14 @@ class IPAddressResource(RalphModelResource):
     class Meta:
         model = networks.IPAddress
 
+    def skip_row(self, instance, original):
```

Comment:
this is built-in method of django-import-export. The purpose of it (in our case) is to skip importing IPAddress if it already exists in database, to, for example, not re-assign it's base object. But - when i'm thinking about it right now, there should be additional logic here - either we should check if `id` column for this ip is filled in csv (purpose: update record, we should pass it) or if this is import from ralph 2.